admin/views/accessibility-statement.ejs

```diff
@@ -44,21 +39,21 @@
             <p class="govuk-body">The MTC does not support third-party screen readers due to compatibility issues with the timed element of the check.</p>
 
             <p class="govuk-body">
-                To address this, MTC provides a number of access arrangements that can be applied to the pupil check. The following
-                table shows which access arrangements are compatible with each device.
+                To address this, there are a number of access arrangements that can be applied to the pupil check.
+                The following table shows which access arrangements are compatible with each device.
             </p>
 
             <table class="govuk-table govuk-spacious govuk-!-font-size-16">
                 <thead class="govuk-table__head">
                 <tr>
                     <th>Access arrangement</th>
-                    <th>Recommended browser/device</th>
+                    <th>Recommended browser and device</th>
                 </tr>
                 </thead>
                 <tbody>
                 <tr>
                     <td>Audible time alert</td>
-                    <td>Not compatible with IE11</td>
+                    <td>Any browser/device</td>
```

Comment:
Technical compatibility.  @GeorgeChatzigiannis - can you confirm this advice is correct?

Reply:
@jon-shipley updated accordingly, thanks for spotting this.